src/typeahead/typeahead.ts

```diff
@@ -187,6 +192,12 @@ export class NgbTypeahead implements ControlValueAccessor,
 
   handleKeyDown(event: KeyboardEvent) {
     if (!this._windowRef) {
+      if (event.which === Key.Enter) {
```

Comment:
I did not want to do it for tab also, since I would still want tab to move focus to the next focusable item.

I'm not sure I would ever really want tab to select an item from the autocomplete. Maybe this is something we can make configurable in the future?